dts/arm/microchip/mec172x/mec172x-vw-routing.dtsi

```diff
@@ -4,111 +4,228 @@
 			vw-reg = <0x41 MSVW 3 1>;
 			vw-girq = <24 13>;
+			status = "okay";
 		};
-		vw-slp-a-n {
+		vw_slp_a_n: vw_slp_a_n {
 			vw-reg = <0x41 MSVW 3 3>;
 			vw-girq = <24 15>;
+			status = "okay";
 		};
-		vw-slp-lan-n {
+		vw_slp_lan_n: vw_slp_lan_n {
 			vw-reg = <0x42 MSVW 4 0>;
 			vw-girq = <24 16>;
+			status = "okay";
 		};
-		vw-slp-wlen-n {
+		vw_slp_wlan_n: vw_slp_wlan_n {
 			vw-reg = <0x42 MSVW 4 1>;
 			vw-girq = <24 17>;
+			status = "okay";
 		};
-		vw-host-c10 {
+		vw_host_c10: vw_host_c10 {
 			vw-reg = <0x47 MSVW 7 0>;
 			vw-girq = <25 0>;
+			status = "okay";
 		};
-		vw-dnx-warn {
+		vw_dnx_warn: vw_dnx_warn {
 			vw-reg = <0x4a MSVW 8 1>;
 			vw-girq = <25 5>;
+			status = "okay";
 		};
 
 		/* Device to Host (SMVW) do not have SoC interrupts */
-		vw-oob-rst-ack {
+		vw_oob_rst_ack: vw_oob_rst_ack {
 			vw-reg = <0x04 SMVW 0 0>;
+			status = "okay";
 		};
-		vw-wake-n {
+		vw_wake_n: vw_wake_n {
 			vw-reg = <0x04 SMVW 0 2>;
+			status = "okay";
 		};
-		vw-pme-n {
+		vw_pme_n: vw_pme_n {
 			vw-reg = <0x04 SMVW 0 3>;
+			status = "okay";
 		};
-		vw-slave-boot-load-done {
+		vw_slave_boot_done: vw_slave_boot_done {
 			vw-reg = <0x05 SMVW 1 0>;
+			status = "okay";
 		};
-		vw-error-fatal {
+		vw_error_fatal: vw_error_fatal {
 			vw-reg = <0x05 SMVW 1 1>;
+			status = "okay";
 		};
-		vw-error-non-fatal {
+		vw_error_non_fatal: vw_error_non_fatal {
 			vw-reg = <0x05 SMVW 1 2>;
+			status = "okay";
 		};
-		vw-slave-boot-load-status {
+		vw_slave_boot_status: vw_slave_boot_status {
 			vw-reg = <0x05 SMVW 1 3>;
+			status = "okay";
 		};
-		vw-sci-n {
+		vw_sci_n: vw_sci_n {
 			vw-reg = <0x06 SMVW 2 0>;
+			status = "okay";
 		};
-		vw-smi-n {
+		vw_smi_n: vw_smi_n {
 			vw-reg = <0x06 SMVW 2 1>;
+			status = "okay";
 		};
-		vw-rcin-n {
+		vw_rcin_n: vw_rcin_n {
 			vw-reg = <0x06 SMVW 2 2>;
+			status = "okay";
 		};
-		vw-host-rst-ack {
+		vw_host_rst_ack: vw_host_rst_ack {
 			vw-reg = <0x06 SMVW 2 3>;
+			status = "okay";
 		};
-		vw-sus-ack-n {
+		vw_sus_ack_n: vw_sus_ack_n {
 			vw-reg = <0x40 SMVW 3 0>;
+			status = "okay";
 		};
-		vw-dnx-ack {
+		vw_dnx_ack: vw_dnx_ack {
 			vw-reg = <0x40 SMVW 3 1>;
+			status = "okay";
+		};
+		vw_t2c_gpio_0: vw_t2c_gpio_0 {
+			vw-reg = <0x50 SMVW 6 0>;
+			status = "disabled";
+		};
+		vw_t2c_gpio_1: vw_t2c_gpio_1 {
+			vw-reg = <0x50 SMVW 6 1>;
+			status = "disabled";
+		};
+		vw_t2c_gpio_2: vw_t2c_gpio_2 {
+			vw-reg = <0x50 SMVW 6 2>;
+			status = "disabled";
+		};
+		vw_t2c_gpio_3: vw_t2c_gpio_3 {
+			vw-reg = <0x50 SMVW 6 3>;
+			status = "disabled";
+		};
+		vw_t2c_gpio_4: vw_t2c_gpio_4 {
+			vw-reg = <0x51 SMVW 7 0>;
+			status = "disabled";
+		};
+		vw_t2c_gpio_5: vw_t2c_gpio_5 {
+			vw-reg = <0x51 SMVW 7 1>;
+			status = "disabled";
+		};
+		vw_t2c_gpio_6: vw_t2c_gpio_6 {
+			vw-reg = <0x51 SMVW 7 2>;
+			status = "disabled";
+		};
+		vw_t2c_gpio_7: vw_t2c_gpio_7 {
+			vw-reg = <0x51 SMVW 7 3>;
+			status = "disabled";
+		};
+		vw_t2c_gpio_8: vw_t2c_gpio_8 {
```

Comment:
@jvasanth1 OCB host index is incorrect as indicated on the documented table. This should be 50h

Reply:
per following 3 information:

////
1:
vw_t2c_gpio_8: vw_t2c_gpio_8 {
vw-reg = <0x52 SMVW 8 0>;
status = "disabled";
};
2:
MCHP_DT_ESPI_VW_ENTRY(ESPI_VWIRE_SIGNAL_OCB_0, vw_t2c_gpio_8),
3:
*  52h   | SMVW08 | ocb_3        | ocb_2        | ocb_1     | ocb_0       |
////

OCB_0 is located at Index 52h bit0.

do you request to move OCB_0 to Index 50h bit0?

Reply:
Porvide one more information from Chrome EC project:

https://chromium-review.googlesource.com/c/chromiumos/platform/ec/+/3863940
__override void board_overcurrent_event(int port, int is_overcurrented)
{
	/*
	 * Meteorlake PCH uses Virtual Wire for over current error,
	 * hence Send 'Over Current Virtual Wire' eSPI signal.
	 */
	espi_send_vwire(espi_dev, port + ESPI_VWIRE_SIGNAL_SLV_GPIO_0,
			!is_overcurrented);
}

ESPI_VWIRE_SIGNAL_SLV_GPIO_0 is used for overcurrent, i.e. VW Index 50h bit0.
	ESPI_VWIRE_SIGNAL_OCB_0,
	ESPI_VWIRE_SIGNAL_OCB_1,
	ESPI_VWIRE_SIGNAL_OCB_2,
	ESPI_VWIRE_SIGNAL_OCB_3,
in driver should be removed and redefined in application level?

Reply:
Yes, From the eSPI spec looks like there is no mention about OCB. OCB is more of a platform specific.
Non Intel SOC with eSPI can use it for some other purpose.

Reply:
Sounds like ESPI_VWIRE_SIGNAL_SLV_GPIO_0 is same than ESPI_VWIRE_SIGNAL_OCB_0 both are intended to be transmitted over current status and used host index 50h. 

So having OCB in 50h and 52h is not ok.

Option #1: 
ESPI_VWIRE_SIGNAL_OCB_0 existed first as platform-specific virtual wire, so I'm thinking we should keep that.

Option #2: Remove ESPI_VWIRE_SIGNAL_OCB_0 altogether from drivers/espi.h 
This breaks current apps.

Option #3: Alternatively we could probably have some map definitions in drivers/espi.h 
#define ESPI_VWIRE_SIGNAL_OCB_0  ESPI_VWIRE_SIGNAL_SLV_GPIO_0 . 
#define ESPI_VWIRE_SIGNAL_OCB_1  ESPI_VWIRE_SIGNAL_SLV_GPIO_1.

I'm inclined for #1 or #3. But in all options this implies to cleanup this initialization table to avoid the duplicate custom virtual wires.

Reply:
cc: @keith-zephyr 

Is there a plan to make ESPI_VWIRE_SIGNAL_OCB_x as VW name in eSPI spec?
if so, we can remove the ESPI_VWIRE_SIGNAL_SLV_GPIO_x names drivers/espi.h and rename them as ESPI_VWIRE_SIGNAL_OCB_x in NPCX code.
else we can remove ESPI_VWIRE_SIGNAL_OCB_x in drivers/espi.h and rename them as ESPI_VWIRE_SIGNAL_SLV_GPIO_x in MCHP code.

Naming VWs at 51h with ESPI_VWIRE_SIGNAL_SLV_GPIO_3 may look odd.

Option 3 at At the application code looks good

Reply:
@vphirema you are missing the main point that is there is duplication of "mappings" in the table for VW that is intended to convey the same info. 

@MartinYan-mchp please let's start by correcting the table. I'm ok if you leave the  ESPI_VWIRE_SIGNAL_SLV_GPIO_x names in said table if this PR includes the change proposed in drivers/espi.h

Reply:
Option 3 looks reasonable

Keep all naming as `ESPI_VWIRE_SIGNAL_SLV_GPIO_x`

and add mapping in drivers/espi.h
```
#define ESPI_VWIRE_SIGNAL_OCB_0 ESPI_VWIRE_SIGNAL_SLV_GPIO_8
#define ESPI_VWIRE_SIGNAL_OCB_1 ESPI_VWIRE_SIGNAL_SLV_GPIO_9
#define ESPI_VWIRE_SIGNAL_OCB_2 ESPI_VWIRE_SIGNAL_SLV_GPIO_10
#define ESPI_VWIRE_SIGNAL_OCB_3 ESPI_VWIRE_SIGNAL_SLV_GPIO_11
```

Reply:
updated